cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -1302,3 +1333,55 @@ with the metadata returned as a JSON string in the return body::
      - The properties requested were returned as a JSON string in the body of the response
    * - ``404 NOT FOUND``
      - The entity, program, or run for which properties are being requested was not found
+
+**Note**: The metadata aggregation of program, application, stream and dataset associated with a program is
+deprecated and program runs are treated like any other entity to which metadata can be associated directly.
+To retrieve the direct metadata of program run please refer to
+:ref:`Metadata Properties <http-restful-api-metadata-properties>` and
+:ref:`Metadata Tags <http-restful-api-metadata-tags>`. An additional query parameter ``aggregateRun`` must be set to
+``false`` while operating with metadata directly associated with program runs.
+
+Metadata for Custom Entities
```

Comment:
OK I see the examples are down here. It would be helpful to link here from above so that people can understand what metadata-details are.

Reply:
Done.